backend/test/routes/zipDownload.js

```diff
@@ -82,10 +82,10 @@ describe('downloading a zip file', () => {
             expect(res.headers['content-disposition'])
               .to.eq('attachment; filename="myShow.zip"', 'file name must be show name')
           })
-          .then(res => JSZip.loadAsync(res.body))
+          .then(res => JSZip.loadAsync(res.body, {createFolders: true}))
           .then(zip => {
             expect(Object.keys(zip.files).length)
-              .to.eq(0, 'should have no files')
+              .to.eq(0, 'Files Present: ' + Object.keys(zip.files) + '\nshould have no files')
```

Comment:
I added some extra output to the test failure text for debugging purposes, but it was pretty helpful, so I figured I'd leave it in since it will only come up if a test fails anyway.